packages/debugger-extension/src/index.ts

```diff
@@ -505,7 +507,7 @@ const main: JupyterFrontEndPlugin<void> = {
 
     commands.addCommand(CommandIDs.debugContinue, {
       label: trans.__('Continue'),
-      caption: trans.__('Continue'),
+      caption: trans.__('Continue (F9)'),
```

Comment:
As an alternative to encoding the keybinding in the caption string, maybe if would also be possible to get the keybinding with the `CommandRegistry.formatKeystroke` method?

The approach would be similar to what is currently used for the simple mode switch in the status bar:

https://github.com/jupyterlab/jupyterlab/blob/d065f9442332f90128a7436561d0f99b6a591107/packages/statusbar-extension/src/index.ts#L370-L381

Reply:
Agreed! thanks for the tip @jtpio :)